src/api/editMessage.js

```diff
@@ -0,0 +1,15 @@
+import { apiPatch } from './apiFetch';
+
+export default async (
+  auth,
+  content,
+  id
+) =>
+  apiPatch(
+    auth,
+    `messages/${id}`,
+    {
+      content
+    },
+    res => res,
```

Comment:
Use this to return a more concrete field if only one is relevant which likely is the case.
Maybe `res.content` or even `res.content.raw_content`?